sphinx_wagtail_theme/layout.html

```diff
@@ -17,7 +17,7 @@
     {%- endif %}
 {%- endif %}
 <!DOCTYPE html>
-<html class="no-js" lang="en">
+<html class="no-js" {% if language is not none %}lang="{{ language }}"{% endif %}>
```

Comment:
why not
```suggestion
<html class="no-js" lang="{{ language|default:"en" }}"{% endif %}>
```

Reply:
That would be a valid decision as well. I am following the documented Sphinx behavior of if no language is selected, "en" is not forced into the document (which may or may not be the correct language). Generally speaking it is nice as a user when sphinx themes match up with the sphinx documentation :)

https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-language

Reply:
Should be easy enough to do with release notes 👍